projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/field-read.html

```diff
@@ -1,4 +1,4 @@
-<div [hidden]="caseField.hidden">
+<div  id="table" [hidden]="caseField.hidden">
```

Comment:
You are going to end up with a lot of fields with the id "table" because this is generic code. Also it's not a table, so it's misleading to use "table". It would be better if you were able to derive the id from some property of the component. Probably the best thing would be to use this.caseField.id()

Reply:
@andywilkinshmcts . Implemented